integration_tests/Rpc.spec.ts

```diff
@@ -1,9 +1,40 @@
 import { SDK } from "../";
-import { H256, SignedParcel, Invoice, AssetMintTransaction, Asset, AssetScheme, AssetTransferTransaction } from "../lib/core/classes";
+import { H256, SignedParcel, Invoice, AssetMintTransaction, Asset, AssetScheme, AssetTransferTransaction, Parcel } from "../lib/core/classes";
 import { PlatformAddress } from "../lib/key/classes";
-import { getAccountIdFromPrivate, generatePrivateKey, signEcdsa } from "../lib/utils";
+import { getAccountIdFromPrivate, generatePrivateKey, signEcdsa, blake256 } from "../lib/utils";
```

Comment:
Why `blake256`?

Reply:
I fixed. It's accidentally added.

We should find a way to automatically check the styles of the tests.